torch/csrc/jit/codegen/cuda/type.cpp

```diff
@@ -168,6 +157,11 @@ static _enum_unordered_map<ParallelType, std::string> parallel_type_string_map{
     {ParallelType::Unroll, "Unroll"},
     {ParallelType::Serial, "Serial"}};
 
+static _enum_unordered_map<MemoryType, std::string> memory_type_string_map{
+    {MemoryType::Local, "register"},
```

Comment:
NITPICK: I get where this is coming from, but is it necessary to have the discrepancy of marking something as `Local` while the intention is `register`.